docker/annovar/README

```diff
@@ -0,0 +1,9 @@
+To build an annovar docker image, first download the annovar tar.gz archive
+from https://annovar.openbioinformatics.org/en/latest/ into this directory.
+
+Next, extract the archive, and ensure that the annovar Perl executables are located
+at annovar/*.pl.
```

Comment:
ah... maybe not

Reply:
Yeah, so Annovar is a bit weird in that to download it you're supposed to go through a (free) registration process, and you get emailed a link to the download. Very archaic! It does look like the links stay active indefinitely though, so I could update the Dockerfile to download the package itself, but it could break at any time.

Reply:
Yeah, maybe that's not worth doing then. We could use `skopeo` to copy the built image directly into our artifactory, so long as there are no licensing implications

Reply:
Could you put the download on a bucket, and then copy it in? I guess that requires creds thought which isn't great.